common/pulp_rpm/common/file_utils.py

```diff
@@ -34,3 +37,12 @@ def calculate_size(file_handle):
     file_handle.seek(0, 2)
     size = file_handle.tell()
     return size
+
+
+def make_packages_relative_path(filename):
+    """Create relative path for package inside repository.
+
+    e.g. "foo.rpm" -> "Packages/f/foo.rpm"
```

Comment:
Use ``param`` and ``returns`` to specify the inputs and outputs. Obvious as it may be, there is value in consistency and readability.